docs/getting-started/detections-req.asciidoc

```diff
@@ -52,49 +54,72 @@ NOTE: For instructions about using Machine Learning jobs and rules, refer to <<m
 
 a|The `manage`, `write`,`read`, and `view_index_metadata` index privileges for the following system indices where `<kib-space>` is the {kib} space name:
 
+* `.alerts-security.alerts-<kib-space>`
 * `.siem-signals-<kib-space>`
 * `.lists-<kib-space>`
 * `.items-<kib-space>`
-|{kib} space `All` privileges for the `Security` feature (see
+
+*NOTE*: If you are upgrading to {stack} 8.0.0, it is recommended that users have privileges to the `.alerts-security.alerts-<kib-space>` AND `.siem-signals-*` index. If you are newly installing the {stack}, then users do not need privileges to the `.siem-signals-*` index.
+
+|{kib} space `All` privileges for the `Security` feature (visit
 {kibana-ref}/xpack-spaces.html#spaces-control-user-access[Feature access based on user privileges])
 
 
 |Enable the Detections feature in all Kibana spaces
 
-*Note*: To turn on the Detections feature, visit the Detections page for each appropriate Kibana space.
+*NOTE*: To turn on the Detections feature, visit the Detections page for each appropriate Kibana space.
 
 |The `manage` privilege
 a|The `manage`, `write`,`read`, and `view_index_metadata` index privileges for the following system indices:
 
+* `.alerts-security.alerts-*`
 * `.siem-signals-*`
 * `.lists-*`
 * `.items-*`
-|{kib} space `All` privileges for the `Security` feature (see
+
+*NOTE*: If you upgrading to {stack} 8.0.0, it is recommended that users have privileges to the `.alerts-security.alerts-<kib-space>` AND `.siem-signals-*` index. If you are newly installing the {stack}, then users do not need privileges to the `.siem-signals-*` index.
+
+
+|{kib} space `All` privileges for the `Security` feature (visit
+{kibana-ref}/xpack-spaces.html#spaces-control-user-access[Feature access based on user privileges])
+
+
+| Preview rules
+|N/A
+a| The `read` privilege to the `.preview.alerts-security.alerts-<kib-space>` index
```

Comment:
```suggestion
a| The `read` privilege for the `.preview.alerts-security.alerts-<kib-space>` index
```
I see both "privileges for" and "privileges to" throughout this page, but since we use "privileges for" in the next line I figured this would seem more consistent.